server/src/main/java/org/opensearch/index/remote/RemoteSegmentTransferTracker.java

```diff
@@ -580,5 +580,53 @@ public int hashCode() {
                 directoryFileTransferTrackerStats
             );
         }
+
+        @Override
+        public String toString() {
```

Comment:
nit: StringBuilder to avoid creating too many strings?

Reply:
Will take this up in a future PR.